src/executables/master.c

```diff
@@ -694,7 +696,7 @@ css_send_to_existing_server (CSS_CONN_ENTRY * conn, unsigned short rid, CSS_SERV
 		    }
 		  else if (!temp->ha_mode)
 		    {
-		      temp = css_return_entry_of_server (server_name, css_Master_socket_anchor);
+		      temp = css_return_entry_of_server (server_name, css_Master_socket_anchor, SERVER_TYPE_ANY);
```

Comment:
```suggestion
		      temp = css_return_entry_of_server (server_name, css_Master_socket_anchor, type);
```

Reply:
Who would type be in this context?

Reply:
scratch this, it is an outdated comment.